osa/provenance/capture.py

```diff
@@ -605,6 +616,7 @@ def get_system_provenance():
             version=platform.python_version(),
             compiler=platform.python_compiler(),
             implementation=platform.python_implementation(),
+            packages=get_python_packages(),
```

Comment:
In which file can I see this python package info in the end?

Reply:
All system info is in the `processlevel_runid_prov.log` files, it is remove from from `json` and `pdf` graphs to improve readability.